_posts/2023-11-23-geospatial-techno.md

```diff
@@ -0,0 +1,76 @@
+---
+author: GeoSpatial Techno
+layout: post
+title: GeoServer installation methods on Windows
+date: 2023-11-23
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [Reddit](https://www.reddit.com/user/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### GeoServer installation methods: “Windows Installer” and “Web Archive”
+GeoServer installation methods: “Windows Installer” and “Web Archive”
+In this session, we will talk about how to install GeoServer software by two commonly methods in Windows. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/G47vV2od4yc).
+
+[![](https://img.youtube.com/vi/G47vV2od4yc/0.jpg)](https://www.youtube.com/watch?v=G47vV2od4yc)
+
+### Introduction
+GeoServer can be installed on different operating systems, since it's a Java based application. You can run it on any kind of operating system for which exists a Java virtual machine. GeoServer’s speed depends a lot on the chosen Java Runtime Environment (JRE). The latest versions of GeoServer are tested with both OracleJRE and OpenJDK. These versions are:
+- Java 17 for GeoServer 2.23 and above
+- Java 11 for GeoServer 2.15 and above
+- Java 8 for GeoServer 2.9 to GeoServer 2.22
+- Java 7 for GeoServer 2.6 to GeoServer 2.8
+- Java 6 for GeoServer 2.3 to GeoServer 2.5
+- Java 5 for GeoServer 2.2 and earlier
+
+There are many ways to install GeoServer on your system. This tutorial will use the Oracle JRE8 also known as JRE 1.8 and discuss the two most commonly used installation methods for windows.
+- Windows Installer
+- Web Archive
+
+### Windows installer
+The Windows installer provides an easy way to set up GeoServer on your system, as it requires no configuration files to be edited or command line settings.
+
+#### Installation
+- Navigate to Oracle.com and download JRE 8 from the Products tab. Consider the operating system architecture and memory requirements when selecting a JRE installer. 32-bit Java version is restricted to 2 GB memory, while the 64-bit version is recommended for optimal server memory. Utilizing JAI with the 32-bit JRE can enhance performance for WMS output generation and raster operations.
```

Comment:
Why JRE 8?

Reply:
Actually we cannot recommend oracle JDK at this time either.

I expect this blog post would do better to link to the online instructions - https://docs.geoserver.org/latest/en/user/installation/index.html#installation ?

Reply:
You are right

Reply:
Nope- oracle does not offer a Java 11 anymore for free (it is too old and they do not official oracle Java anymore for production use unless you pay). 

We recommend OoenJDK (included with your Linux distribution) or from Adoptium 

Please review the current install instructions and update your blog post.